apps/frontend/src/components/proposal/ProposalParticipant.tsx

```diff
@@ -55,7 +55,9 @@ export default function ProposalParticipant(props: {
           label="Principal Investigator"
           value={
             props.principalInvestigator
-              ? `${props.principalInvestigator.firstname} ${props.principalInvestigator.lastname}; ${props.principalInvestigator.organisation}`
+              ? `${props.principalInvestigator.preferredname}`
```

Comment:
Maybe it is good idea to extract this in a function to be more clear what is it about. For example `getFullUserNameWithOrganisation` something similar as `getFullUserNameWithEmail`.